x-pack/platform/plugins/shared/lens/public/app_plugin/shared/edit_on_the_fly/flyout_wrapper.tsx

```diff
@@ -113,8 +113,13 @@ export const FlyoutWrapper = ({
                 </EuiText>
               </EuiFlexItem>
             )}
+            <EuiFlexItem grow={true} />
             {toolbar && (
-              <EuiFlexItem grow={false} data-test-subj="lnsVisualizationToolbar">
+              <EuiFlexItem
+                css={css({ zIndex: euiTheme.levels.menu })}
```

Comment:
Works fine, but it looks strange having a zIndex here as this is the "space" for the toolbar not actually the flyout (even if is contained within that button.
Do you think we can try to move it elsewhere like in the FlyoutContainer like here https://github.com/mariairiartef/kibana/blob/77dbdac93d8e3e337e04c83450fcc8502a7ae07f/x-pack/platform/plugins/shared/lens/public/shared_components/flyout_container.tsx#L106-L116
Seems more appropriate, but not sure if you already have evaluated that and if it fine